### ISSUE TITLE
many_parentheses, not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2639]
https://rules.abaplint.org/many_parentheses/

```abap
LOOP AT tab_statements ASSIGNING <fs_stmnt_tmp> WHERE ( type <> 'P' AND type <> 'S' AND type <> 'G' ) AND from <= limit.
ENDLOOP.
```
